src/helm-frontend/src/services/getInstances.ts

```diff
@@ -1,18 +1,32 @@
 import Instance from "@/types/Instance";
 import getBenchmarkEndpoint from "@/utils/getBenchmarkEndpoint";
-import getVersionBaseUrl from "@/utils/getVersionBaseUrl";
+import getBenchmarkSuite from "@/utils/getBenchmarkSuite";
 
 export default async function getInstancesByRunName(
   runName: string,
   signal: AbortSignal,
+  suite?: string,
 ): Promise<Instance[]> {
   try {
-    const instances = await fetch(
-      getBenchmarkEndpoint(`${getVersionBaseUrl()}/${runName}/instances.json`),
-      { signal },
-    );
+    if (suite) {
+      const instances = await fetch(
+        getBenchmarkEndpoint(
+          `/benchmark_output/runs/${suite}/${runName}/instances.json`,
```

Comment:
I think the same change (regarding removing if-else) requested above also applies to these other files (or am I missing something)?

Reply:
Fixed all if-else blocks like this